### PR TITLE
 [FIX] Link: properly apply classes on link

### PR DESCRIPTION
**Pull request by Zynton**
Linked Jabberwock PR: https://github.com/odoo-dev/jabberwock/pull/203

### CODE REVIEW

File: addons/web_editor/static/src/js/wysiwyg/wysiwyg.js [hunk 1]
```diff
@@ -129,9 +129,9 @@ var Wysiwyg = Widget.extend({
                 snippetMenuElement: $mainSidebar[0],
                 snippetManipulators: $snippetManipulators[0],
                 customCommands: {
-                    openMedia: { handler: this.openMediaDialog.bind(this) },
-                    openLinkDialog: { handler: this.openLinkDialog.bind(this) },
-                    saveOdoo: { handler: this.saveToServer.bind(this) }
+                    openMedia: {handler: this.openMediaDialog.bind(this)},
+                    openLinkDialog: {handler: this.openLinkDialog.bind(this)},
+                    saveOdoo: {handler: this.saveToServer.bind(this)}
```
Comment (dmo-odoo) [inline addons/web_editor/static/src/js/wysiwyg/wysiwyg.js:138]: The previous verion was better IMO, how is the linter prefering the new version ?

**Comment by dmo-odoo** [comment]
I'm not sure we should use our project's eslint/prettier config on Odoo's js files ?

File: addons/web_editor/static/src/js/wysiwyg/wysiwyg.js (hunk 1 above)
Comment (Zynton) [reply addons/web_editor/static/src/js/wysiwyg/wysiwyg.js:138]: Uh that's strange indeed. Now `prettier` saves it automatically like this and I can't save the old version so maybe you could do this for me while I figure out the configuration issue?

Comment (dmo-odoo) [reply addons/web_editor/static/src/js/wysiwyg/wysiwyg.js:138]: Actually you may be right. There seems to be something weird with our linter. I thought it was supposed to only apply to ts files but I just noticed it is warning me for odoo js files as well.

**Pull request merged**